core/SCsub

```diff
@@ -97,6 +97,20 @@ thirdparty_minizip_sources = ["ioapi.c", "unzip.c", "zip.c"]
 thirdparty_minizip_sources = [thirdparty_minizip_dir + file for file in thirdparty_minizip_sources]
 env_thirdparty.add_source_files(thirdparty_obj, thirdparty_minizip_sources)
 
+# Tiny-process-library
+thirdparty_tiny_process_library_dir = "#thirdparty/tiny-process-library/"
+thirdparty_tiny_process_library_sources = []
+
+if env["platform"] in ["linuxbsd", "macos", "web", "android"]:
```

Comment:
I don't think the library do anything on the web, so we should exclude the third party library there I guess.